### PR TITLE
Adding changelog

### PR DESCRIPTION
**Pull request by nv-braf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Changelog
+
+## 23.06 (2023-06-29)
```
Comment (tgerdesnv) [inline CHANGELOG.md:19]: We need to be using MA version, not DLFW release number (although we could also include DLFW information)

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- `request-rate-range` can now be searched in brute mode
+- Capture PA errors in a log file
+- Added detection for Triton Server launch failures
+- Added `cpu_only` option for ensemble composing models
+- Added binary concurrency search to quick search mode
+- Added binary parameter search to brute search mode
+
+## 23.04 (2023-04-27)
+
+- Support for BLS models added
```
Comment (tgerdesnv) [inline CHANGELOG.md:37]: does this support links? Can we link docs for new features?

File: CHANGELOG.md (hunk 1 above)
Comment (nv-braf) [reply CHANGELOG.md]: Understood and fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (nv-braf) [reply CHANGELOG.md]: It does, and I've added them to the relevant bullet points

**Review by tgerdesnv — APPROVED** [review]

**Pull request merged**